src/formatter/helpers/issue_helpers.js

```diff
@@ -171,3 +173,15 @@ export function formatIssue({
   })
   return `${text}\n`
 }
+
+function getRetryWarningText(testCase, flakyColorFn) {
+  const result = testCase.result
+  if (!testCase.result) {
+    return ''
+  }
+  const attemptNumber = testCase.attemptNumber
+  if (attemptNumber > 1 || result.status === Status.RETRY) {
+    return flakyColorFn(`(attempt #${attemptNumber}) `)
```

Comment:
Thoughts on having this be `(attempt X, retried)`

Reply:
Are you proposing:

`` return flakyColorFn(`(attempt ${attemptNumber}, retried) `) ``

?

Reply:
I think he just want to specifically mention that the test has been retried, despite "attempt #x" already posing so.
Maybe we can have

    return flakyColorFn(`(test retried, attempt #${attemptNumber})`)

Reply:
@chipironcin your code provides the same information as mine, just inverting the order ;)